nav-app/src/app/viewmodels.service.ts

```diff
@@ -369,6 +369,7 @@ export class ViewModel {
 
 
     hideDisabled: boolean = false; //are disabled techniques hidden?
+    showAllSubtechniques: boolean = false;
```

Comment:
Setting this boolean with the control doesn't actually _do_ anything, and it isn't the expected functionality of the control. 
- Clicking the button should show/hide all sub-techniques, but this boolean isn't referenced anywhere from the Matrix layouts to satisfy that requirement. It's only ever referenced to set the value when the user clicks the toggle button.
- There probably shouldn't be a single boolean value to override the "opened" state for all parent techniques. My intention with this feature was that it'd set the `showSubtechniques` field for every TechniqueVM, therefore affecting exported layers and so forth as well. It'd essentially be a shortcut to the UX of clicking the sidebar on every technique.